DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/_DelegateProgressSummary.cshtml

```diff
@@ -58,8 +60,8 @@
     </dt>
     <partial name="_SummaryFieldValue" model="Model.Completed" />
     <dd class="nhsuk-summary-list__actions">
-      <a href="#">
-        Edit<span class="nhsuk-u-visually-hidden"> completion date</span>
+      <a href="/TrackingSystem/Delegates/@DelegateProgressAccessRoute.CourseDelegates/DelegateProgress/@Model.ProgressId/EditCompletionDate">

```

Comment:
Is there a reason we aren't using the asp-controller, asp-action etc for this?

Reply:
nope, changed now